### PR TITLE
Add styling to LearningObjects listing page 2

### PR DESCRIPTION
**Pull request by pwilkins**
Add styling to LORE listing page, implementing designer's templates.
This is an initial pass at styling the page since the finished design
requires features that are not yet implemented.

Please review - @ferdi @pdpinch @carsongee

Closes Issue #73

### CODE REVIEW

File: ui/templates/listing.html [hunk 1]
```diff
@@ -1,82 +1,104 @@
 {% extends "base.html" %}
 {% block content %}
-<div id="header">
-<h1>Listing</h1>
-</div>
+<div id="header"></div>
 
 <div>
     <a href="{% url 'upload' repo.id %}">import</a>
 </div>
```
Comment (pdpinch) [inline ui/templates/listing.html:7]: This link and block should be removed. It doesn't belong here.

File: ui/templates/base.html [hunk 2]
```diff
@@ -62,8 +62,8 @@
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
           <ul class="nav navbar-nav">
             {% block nav %}{% endblock %}
-            <!-- <li><a href="#"><i class="fa fa-arrow-circle-o-right"></i> Import Course</a></li> -->
-            <!-- <li><a href="#0" class="btn-taxonomies"><i class="fa fa-cogs"></i> Manage Taxonomies</a></li> -->
+            <li><a href="#"><i class="fa fa-arrow-circle-o-right"></i> Import Course</a></li>
```
Comment (pdpinch) [inline ui/templates/base.html:65]: This should link to the the upload form:

`<a href="{% url 'upload' repo.id %}">`

Perhaps?

Comment (pdpinch) [inline ui/templates/base.html:65]: Nope. That's not it. @ShawnMilo what should this be now?

Comment (ShawnMilo) [inline ui/templates/base.html:65]: Ahh. If the slug branch has been merged, then probably `repo.slug` instead of `repo.id`.

@pdpinch

**Comment by carsongee** [comment]
Mind if I close this, now that #85 is up?

**Comment by pdpinch** [comment]
yes please -- to avoid confusion.

**Pull request closed without merging**